### PR TITLE
Add Utility for Creating SQLite Database with USPS Data

### PR DESCRIPTION
**Pull request by dantecoupet**
## PR Description
ticket: https://jira.cms.gov/browse/WNMGGEO-4970

## PR Checklist
* [ ] **New automated tests have been written to the extent possible.**
* [ ] **The code has been checked for structural/syntactic validity.**
	- AMI/application: a build was performed
	- terraform changes: "terraform plan" checked on every affected environment
* [ ] **(If applicable) the code has been manually tested on our infrastructure.**
	- AMI/application: deployed an a test or dev environment
	- terraform changes: applied to test or dev environment
	- script: run against test or dev environment
* [ ] **Likely failure points and new functionality have been identified and tested manually.**
	Examples:
	- Application manually run in a way that triggers any new branches
	- AMI logged into and changes verified from login shell
* [ ] **Pull request description includes a description of all the manual steps performed to accomplish the above.**

To provide feedback on this template, visit https://docs.google.com/document/d/1YfTv7Amyop5G_8w1c2GJ_Mu-70L0KkZHhm9f9umDi3U/edit

### CODE REVIEW

File: cmd/seed-db/main.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+			for i := 0; i < len(citystateData); i++ {
+				params := getParameters(citystateData[i])
+				_, err = tx.Exec(citystateInsertQuery, params...)
+				if err != nil {
+					panic(err)
+				}
+			}
+
+			citystateData = []citystate.CityStateDetail{}
+		}
```
Comment (dacut) [inline cmd/seed-db/main.go]: The if condition should be:
`if len(citystateData) >= BATCH_SIZE`

There's no need for the data to be exactly a batch size to do this insert, unless I'm missing something.

File: cmd/seed-db/main.go [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+	err = tx.Commit()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func getParameters(detail interface{}) []interface{} {
```
Comment (dacut) [inline cmd/seed-db/main.go]: Change occurrences of `interface{}` to `any`. The `any` keyword was added in go1.18.

**Review by dacut — CHANGES_REQUESTED** [review]
A few changes, I think.

File: cmd/seed-db/main.go (hunk 1 above)
Comment (dantecoupet) [reply cmd/seed-db/main.go]: Done

File: cmd/seed-db/main.go (hunk 2 above)
Comment (dantecoupet) [reply cmd/seed-db/main.go]: Done

**Review by dacut — APPROVED** [review]
LGTM!

**Pull request merged**